homeassistant/components/kodi/media_player.py

```diff
@@ -636,11 +636,6 @@ def media_album_artist(self):
 
         return None
 
-    @property
-    def available(self):
-        """Return True if entity is available."""
-        return not self._connect_error
-
     async def async_turn_on(self):
```

Comment:
We are we firing events from an entity ?

Reply:
I'm not sure, I just removed the available method as, once the network connection is lost with Kodi, it causes it to go into unavailable, which doesn't work well for Kodi. Is it required for something related to events, shall I put the method back in, but always return `True`? (for reference, the method wasn't there before, then got added with the implementation that I removed so I would imagine it should be fine to remove it instead of returning always `True`)

Reply:
This is a separate issue.  It really shouldn't work like this, but it does.

Anyways its definitely going to break with adding `available` since you can never turn it on once its unavailable.

Let's revert the change for now.  Hopefully this can be refactored to be a bit more inline with standards in the future.